src/components/Login/Login.jsx

```diff
@@ -103,11 +103,14 @@ export class Login extends React.Component {
             (!this.state.loggedIn) ?
                 <div className={styles.loginPage}>
                      <div className={styles.foo}>
-                        <span className={styles.letter} data-letter="D">D</span>
-                        <span className={styles.letter} data-letter="i">i</span>
-                        <span className={styles.letter} data-letter="X">X</span>
-                        <span className={styles.letter} data-letter="i">i</span>
-                        <span className={styles.letter} data-letter="t">t</span>
+                        <span className={styles.letter} data-letter="T">T</span>
+                        <span className={styles.letter} data-letter="e">e</span>
+                        <span className={styles.letter} data-letter="l">l</span>
+                        <span className={styles.letter} data-letter="l">l</span>
+                        <span className={styles.letter} data-letter="T">T</span>
+                        <span className={styles.letter} data-letter="a">a</span>
+                        <span className={styles.letter} data-letter="l">l</span>
+                        <span className={styles.letter} data-letter="e">e</span>
```

Comment:
If you're trying to avoid Copyright, Telltale is a pretty big [games brand](https://telltale.com/).

Reply:
Oh, this might also look nicer if you didn't write out each `span` but had a map instead.

Reply:
that's a shame, will look at other names

Reply:
done the map, changed the name to 'Telltales' ... hoping that 's' makes all the difference?